docs/_articles/en/einstein/einstein-overview.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Einstein for Developers Overview
+lang: en
+---
+
+Einstein for Developers is an AI-powered developer tool that is available as an easy-to-install VS Code extension built using CodeGen, Salesforce’s secure, custom AI model. Use the tool to enhance developer productivity by quickly generating code suggestions using natural language instructions. The extension is available in the [VS Code](https://marketplace.visualstudio.com/vscode) and [Open VSX](https://open-vsx.org/) marketplaces.
+
+Einstein for Developers assists you throughout the Salesforce development process with expertise learned from anonymized code patterns. The goal of our suite of AI-powered developer tools are to increase productivity and provide helpful assistance for complex coding tasks. We enforce development best practices with our code generation and just-in-time code completion features. AI-assisted tooling also makes it easier for new developers to onboard to the Salesforce Platform.
+
+## Current Capabilities
+
+This Einstein for Developers(beta) focuses on natural language to Apex code generation. This feature, used along with IntelliSense, makes Apex development tooling in Visual Studio Code even richer. Familiarity with Visual Studio Code is assumed.
+
+Currently, you can use the extension to:
+
+- Generate code from natural language instructions, in a side bar. You can then copy the generated code and paste it in a file in the editor as appropriate.
+- Generate code from natural language instructions within an existing Apex class, trigger, or anonymous Apex file. A new command in the VS Code Command Palette, `Einstein: Generate Code`, lets you enter in a prompt describing what you'd like to build and then generate Apex code within your editor.
+
```

Comment:
I think we should reframe this section to focus on how we currently support boilerplate code generation with natural language prompts. There's two modalities that we offer this through - one through the command palette if you'd like to do a quick generation and otherwise through the sidebar if you're looking to continue working with the tool side by side

Reply:
done.